src/main/java/org/alliancegenome/curation_api/constants/VocabularyConstants.java

```diff
@@ -69,14 +69,17 @@ private VocabularyConstants() {
 
 	public static final String GENE_EXPRESSION_VOCABULARY = "gene_expression";
 	public static final String GENE_EXPRESSION_RELATION_TERM = "is_expressed_in";
-	
+
 	public static final String TRANSCRIPT_RELATION_VOCABULARY = "transcript_relation";
 	public static final String TRANSCRIPT_CHILD_TERM = "is_child_of";
 	public static final String TRANSCRIPT_PARENT_TERM = "is_parent_of";
 
 	public static final String STAGE_UBERON_SLIM_TERMS = "stage_uberon_slim_terms";
 	public static final String ANATOMICAL_STRUCTURE_UBERON_SLIM_TERMS = "anatomical_structure_uberon_slim_terms";
-	public static final String CELLULAR_COMPONENT_QUALIFIERS = "cellular_component_qualifiers";
+	public static final String SPATIAL_EXPRESSION_QUALIFIERS = "spatial_expression_qualifiers";
```

Comment:
Can we add _VOCABULARY or _VOCABULARY_TERM_SET suffixes to these names to make it clear what these constants refer to